app/src/pages/admin/serverSettingsPage/serverSettingsTabs/statisticsTab/statisticsTab.jsx

```diff
@@ -138,7 +138,11 @@ export class StatisticsTab extends Component {
   };
 
   toggleStatistics = () => {
-    this.props.tracking.trackEvent(ADMIN_SERVER_SETTINGS_PAGE_EVENTS.MAKE_RP_GREAT_AGAIN);
+    !this.state.statisticsEnabled
```

Comment:
Why not to use just `this.state.statisticsEnabled` and swap the tracking function calls?)

```
this.state.statisticsEnabled
      ? this.props.tracking.trackEvent(ADMIN_SERVER_SETTINGS_PAGE_EVENTS.MAKE_RP_GREAT_AGAIN_UNCHECK)
      : this.props.tracking.trackEvent(
          ADMIN_SERVER_SETTINGS_PAGE_EVENTS.MAKE_RP_GREAT_AGAIN_CHECK,
        );
```